SmileNLP/src/main/java/smile/nlp/SimpleCorpus.java

```diff
@@ -234,23 +234,37 @@ public int getBigramFrequency(Bigram bigram) {
 
     @Override
     public Iterator<Text> search(String term) {
-    	ArrayList<Text> hits = new ArrayList<Text>(invertedFile.get(term));
-        return hits.iterator();
+        if (invertedFile.containsKey(term))
```

Comment:
Our code format is 

if () {
}

I will merge the request and revise it by myself. Please use the same format next time. Thank you very much!